src/serialization/sb3.js

```diff
@@ -684,7 +684,7 @@ const parseScratchObject = function (object, runtime, extensions, zip) {
     const blocks = new Blocks();
 
     // @todo: For now, load all Scratch objects (stage/sprites) as a Sprite.
-    const sprite = new Sprite(blocks, runtime);
+    const sprite = new Sprite(blocks, runtime, object.isStage);
```

Comment:
Instead of adding the `isStage` ctor arg and corresponding property on all `Sprite` objects, what do you think about adding a little bit of code here (and on the SB2 side) to switch the sprite's layer group if `object.isStage` is true?
I guess that might require an API change where calling `setDrawableOrder` without specifying a layer group keeps a sprite in its existing layer group, but that doesn't sound like a bad thing to me.